demo/api/src/products/entities/product.entity.ts

```diff
@@ -95,70 +81,108 @@ export class Product extends BaseEntity<Product, "id"> {
         sort: true, //default is true
         input: true, //default is true
     })
+    @CsvColumn("title")
+    @IsString()
```

Comment:
I'm unsure how this will work with the API Generator, which take validators defined on the entity and adds them to the input.

Reply:
Another Pro for the DTO